lib/features/home/home_screen.dart

```diff
@@ -1,12 +1,17 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:flutter_modular/flutter_modular.dart';
 
 class HomeScreen extends StatelessWidget {
   const HomeScreen({super.key});
 
   @override
   Widget build(BuildContext context) {
+    final localization = AppLocalizations.of(context)!;
```

Comment:
We have a neat extenstion for this:
```
extension Localization on BuildContext {
  AppLocalizations get localizations => AppLocalizations.of(this)!;
}
 ```

Reply:
Then, inside the code we use it like this:
`context.localizations.something`

Reply:
Alright let me change that real quick

Reply:
I wasn't sure where to actually put the extension file so I figured it would fit in the l10n directory along with the .arb files, correct me if I'm wrong

Reply:
Nvm, moved it to core/extensions/localization now, seems much more fitting now

Reply:
Yes, you are correct. core/extensions/ is a good place for it